yt_dlp/extractor/zee5.py

```diff
@@ -114,3 +114,45 @@ def _real_extract(self, url):
             'episode_number': int_or_none(try_get(json_data, lambda x: x['index'])),
             'tags': try_get(json_data, lambda x: x['tags'], list)
         }
+
+
+class Zee5SeriesIE(InfoExtractor):
+    _VALID_URL = r'(?:https?://)(?:www\.)?zee5\.com/(?:global/)?(?:[^#/?]+/)?(?:(?:tvshows|kids)/[^#/?]+/)(?P<display_id>[^#/?]+)/(?P<id>[^#/?]+)/?(?:$|[?#])'
+    _TEST = {
+        'url': 'https://www.zee5.com/kids/kids-shows/krishna-balram/0-6-1871',
+        'playlist_mincount': 43,
+        'info_dict': {
+            'id': '0-6-1871',
+        },
+    }
+
+    def _entries(self, show_id):
+        video_ids = []
+        access_token_request = self._download_json(
+            'https://useraction.zee5.com/token/platform_tokens.php?platform_name=web_app',
+            show_id, note="Downloading access token")
+        headers = {
+            'X-Access-Token': access_token_request['token'],
+            'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.72 Safari/537.36',
```

Comment:
unless a specific user agent is needed, dont set it

Reply:
Yup, it's definitely needed. Else SOMETIMES u get a token missing or invalid token response from the api.

Reply:
yt-dlp automatically adds a valid user agent. From my testing, this seems to be enough. Please confirm

Reply:
yes you are right, it's working.

Reply:
I removed it